packages/expo-modules-autolinking/scripts/ios/autolinking_manager.rb

```diff
@@ -94,12 +96,12 @@ class AutolinkingManager
     end
 
     public def links_for_testing?
-      @options.fetch(:testsOnly, false)
+      return @options.fetch(:testsOnly, false) || @options.fetch(:includeTests, false)
```

Comment:
Do we even need `links_for_testing` anymore? You replaced it above

Reply:
Hah, you're right 😅 I'll remove it then.